README.md

```diff
@@ -0,0 +1,57 @@
+
+If any image in your workload is not signed where trust is enforced, or is not signed by the correct role when a signer is defined, the entire workload is prevented from deploying.
+
+Portieris receives AdmissionRequests for creation of or edits to all types of workload. To prevent Portieris from impacting auto-recovery, it approves requests where a parent exists.
+
+Portieris' Admission Webhook is configured to fail closed. Three instances of Portieris make sure that it is able to approve its own upgrades and auto-recovery. If all instance of Portieris are unavailable, Kubernetes will not auto-recover it, and you must delete the MutatingAdmissionWebhook to allow Portieris to recover.
+
+## Installing Portieris
+
+Portieris is installed using a Helm chart. Before you begin, make sure that you have Kubernetes 1.9 or above, and Helm 2.8 or above installed in your cluster.
+
+To install Portieris:
+
+* Clone the Portieris Git repository to your workstation.
+* Change directory into the Portieris Git repository, then run `helm install -n portieris helm/portieris`.
+
+## Uninstalling Portieris
+
+You can uninstall Portieris at any time using `helm delete --purge portieris`. Note that all your image security policies are deleted when you uninstall Portieris.
+
+## Image security policies
+
+Image security policies define Portieris' behavior in your cluster. There are two types of policy:
+
+* ImagePolicies can be configured in each Kubernetes namespace, and define Portieris' behavior in that namespace. If an ImagePolicy exists in a namespace, the policies from that namespace are used, even if the ImagePolicy does not have a matching policy for a given image. If a namespace does not have an ImagePolicy, the ClusterImagePolicy is used.
+* ClusterImagePolicies are configured at the cluster level, and take effect whenever there is no ImagePolicy in the namespace where the workload is being deployed.
+
+## Configuring image security policies
+
+You can configure custom security policies to control what images can be deployed in your Kubernetes namespaces, and to enforce trust pinning of particular signers. For more information, see the [IBM Cloud docs](https://console.bluemix.net/docs/services/Registry/registry_security_enforce.html#customize_policies).
+
+## Configuring access controls for your security policies
+
+You can configure Kubernetes RBAC rules to define which users and applications have the ability to modify your security policies. For more information, see the [IBM Cloud docs](https://console.bluemix.net/docs/services/Registry/registry_security_enforce.html#assign_user_policy).
+
+## Reporting security issues
+
+To report a security issue, DO NOT open an issue. Instead, send your report via email to alchreg@uk.ibm.com privately.
+
+## Current limitations
```

Comment:
This should be further up, either above or below how it works.

We should also say that we are happy to work with you to support other Notary server implementations.